springerlink.md

```diff
@@ -1,163 +1,1552 @@
+<ul>
+<li>9.2.9 (Web)</li>
+<li>10.2.9 (non-web document)</li>
+<li>11.2.1.9 (Software)</li>
+<li>11.2.2.9 (Closed Software) – Does not apply</li>
+<li>11.6.2 (Authoring Tool)</li>
+<li>12.1.2 (Product Docs)</li>
+<li>12.2.4 (Support Docs)</li>
+</ul>
+<em>Revised Section 508</em>
+<ul>
+<li>501 (Web)(Software)</li>
+<li>504.2 (Authoring Tool)</li>
+<li>602.3 (Support Docs)</li>
+</ul></td>
+<td>Partially Supports</td>
+<td>
+The pages found at link.springer.com generally do not depend solely on sensory characteristics such as shape, size, visual location, orientation, or sound to understand content.
+<br><br>
+Icons may be displayed to control functionality. Adjacent visible text labels or aria-labels may also provided to ensure that relationships are available to Assistive Technologies. 
+<br><br>
+Some icons (e.g. "next" and "previous" pagination controls on search results pages) in the <strong>older</strong> design (currently Journals, Book Series, and ancillary pages such as search, indexes, and account management) lack accessible names. Users who are unable to perceive these icons by their sensory characteristics may be unable to use these controls. We are in the process of transferring all content from the older design to the newer design, which will remove the reliance on sensory characteristics for understanding. 
+<br><br>
+Some complex images in journals, books, series, protocols, reference works and proceedings may rely on sensory characteristics to be understood. Examples may include images of charts, graphs, diagrams, or maps. Generally, these images are used to supplement text content. 
+<br><br>
+Images found at checkout.springer.com that are used to convey meaning (e.g. images to signify which credit cards we accept) are not supplemented with text and cannot be perceived by users of Assistive Technology. 
+<br><br>
+The Freshdesk customer services widget uses background images without text equivalents to initiate functionality. We are actively working to solve this problem. Additionally, we have provided alternate means of accessing customer services on our "Contact Us" page, including a contact email address, a support ticket system, and helpdesk telephone numbers.
+</td>
+</tr>
+<tr id="use-of-color" valign="top">
+<td><a href="http://www.w3.org/TR/WCAG20/#visual-audio-contrast-without-color"><strong>1.4.1 Use of Color</strong></a> (Level A)
+<br><br>Also applies to:<br><br>
+<em>EN 301 549 Criteria</em>
+<ul>
+<li>9.2.10 (Web)</li>
+<li>10.2.10 (non-web document)</li>
+<li>11.2.1.10 (Software)</li>
+<li>11.2.2.10 (Closed Software) – Does not apply</li>
+<li>11.6.2 (Authoring Tool)</li>
+<li>12.1.2 (Product Docs)</li>
+<li>12.2.4 (Support Docs)</li>
+</ul>
+<em>Revised Section 508</em>
+<ul>
+<li>501 (Web)(Software)</li>
+<li>504.2 (Authoring Tool)</li>
+<li>602.3 (Support Docs)</li>
+</ul></td>
+<td>Partially Supports</td>
+<td>
+Colour alone is generally not used to convey information in the pages of this product.
```

Comment:
There is an inconsistency between the use of colour and color in document.

Reply:
Urgh, yeah. I think I'm going to leave these as they are, because the inconsistency comes from the actual source standards. EN 301 549 uses "colour" and Section 508 and WCAG both use "color", so we'll never get complete consistency in the document while we cover all three!

Reply:
I noticed a mixture of `http` and `https` links, this also looks like something from the source standards.